src/Tribe/RSVP.php

```diff
@@ -2315,11 +2349,15 @@ public function parse_attendee_details() {
 	 *             any other case.
 	 */
 	public function parse_ticket_quantity( $product_id ) {
-		if ( empty( $_POST[ "quantity_{$product_id}" ] ) ) {
-			return 0;
+		$quantity = 0;
+
+		if ( isset( $_POST['tribe_ticket_quantity'] ) ) {
+			$quantity = absint( $_POST['tribe_ticket_quantity'] );
+		} elseif ( isset( $_POST[ "quantity_{$ticket_id}" ] ) ) {
+			$quantity = absint( $_POST["quantity_{$ticket_id}"] );
```

Comment:
* :no_entry_sign: Processing form data without nonce verification.
* :no_entry_sign: Array keys must be surrounded by spaces unless they contain a string or an integer.
* :no_entry_sign: There must be a single space after the first square bracket
* :no_entry_sign: There must be a single space before the last square bracket

<sub>hash: b8899077bc48d80c024ca9df1b905787</sub>